src/d3d11/d3d11_main.cpp

```diff
@@ -14,7 +14,39 @@ namespace dxvk {
   
 extern "C" {
   using namespace dxvk;
-  
+
+  //
+  // Workaround MinGW symbol name issues that prevent using DXVK in Wine with a 32-bit
+  // mod that works by proxying dxgi.dll.
+  //
+  // If a linker-dependency on DXVK's dxgi.dll were used, then it would require DXGI.dll
+  // to export "CreateDXGIFactory1@8" and Windows mod DLLs do not do this.
+  //
+  //  Mods can make themselves compatible with old versions of DXVK by adding this code:
+  // 
+  //   #ifdef _M_IX86
+  //   #pragma comment (linker, "/export:CreateDXGIFactory1@8=_CreateDXGIFactory1@8")
+  //   #endif
+  //
+  // To support unaltered 32-bit game mods, DXVK must load the DLL manually...
+  //
+  HRESULT __stdcall CreateDXGIFactory1_Import(REFIID riid, void **ppFactory) {
+#ifdef _M_IX86
```

Comment:
I'd rather just have `#ifdef _WIN32` here and only use the direct call for native builds. There shouldn't be any harm in doing this on 64-bit Windows builds, but if for whatever reason we ever run into an issue here i'd rather not have two different code paths.

Reply:
Fair. I was more interested in just applying the fix to 32-bit builds, since they're the ones affected... but that wasn't accounting for native builds that are also 32-bit but not Windows.

Reply:
nevermind, see above.